scaladoc/src/dotty/tools/scaladoc/tasty/TastyParser.scala

```diff
@@ -134,6 +155,24 @@ case class ScaladocTastyInspector()(using ctx: DocContext) extends DocTastyInspe
       basePck.withMembers((basePck.members ++ rest).sortBy(_.name))
     }.toList -> rootDoc
 
+  def mergeAnyRefAliasAndObject(parser: TastyParser) =
+    val defn = ctx.compilerContext.definitions
+    val oM = parser.parseClasslike(defn.ObjectClass.asInstanceOf[parser.qctx.reflect.Symbol].tree.asInstanceOf[parser.qctx.reflect.ClassDef])
+    val aM = parser.parseTypeDef(defn.AnyRefAlias.asInstanceOf[parser.qctx.reflect.Symbol].tree.asInstanceOf[parser.qctx.reflect.TypeDef])
+    "scala" -> oM.copy(
+      name = aM.name,
+      // dri = aM.dri,
```

Comment:
For now we preserve Object DRI, because AnyRef is a type and is overriding default page with it's content

Reply:
Probably we need to add pages for types or just keep AnyRef as alias to Object